examples/typescript-backbone/js/app.ts

```diff
@@ -339,6 +396,20 @@ class AppView extends Backbone.View {
 		Todos.each(this.addOne);
 	}
 
+	// Filter out completed/remaining tasks
+	filter(criteria: string) {
+		taskFilter = criteria;
```

Comment:
After thinking about this a bit more, instead of a global var, how would you feel about attaching this "State" to the router and when we need to get the current state of the filter just ask the router via a getter* method?